src/app/services/models/annotation.resolver.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from "@angular/core";
+import { ActivatedRouteSnapshot, Resolve } from "@angular/router";
+import { BawSessionService } from "@baw-api/baw-session.service";
+import { SecurityService } from "@baw-api/security/security.service";
+import { AnnotationSearchParameters } from "@components/annotations/pages/annotationSearchParameters";
+import { lastValueFrom } from "rxjs";
+
+@Injectable({ providedIn: "root" })
+class AnnotationsResolver
```

Comment:
naming: this is specifically a _annotation search_ or filter resolver

Reply:
<img width="687" height="97" alt="image" src="https://github.com/user-attachments/assets/220b29ea-9439-4df2-a960-0ef0917aef0e" />